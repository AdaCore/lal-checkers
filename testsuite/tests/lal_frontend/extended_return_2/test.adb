function Ex1 return Integer is
begin
   return X : Integer := 3 do
      X := X + 1;
   end return;
end Ex1;