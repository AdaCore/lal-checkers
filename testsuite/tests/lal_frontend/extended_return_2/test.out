Program:
  X#1 = 3
  X#1 = +(X#1, 1)
  result0#2 = X#1
  goto end
  end: